src/configure/helper/templateHelper.ts

```diff
@@ -41,11 +41,42 @@ export async function mergingRepoAnalysisResults(repoPath: string, repositoryPro
     }
     return analysisResult;
 }
-export async function analyzeRepoAndListAppropriatePipeline(repoPath: string, repositoryProvider: RepositoryProvider, repoAnalysisParameters: RepositoryAnalysisParameters, targetResource?: GenericResource): Promise<PipelineTemplate[]> {
+
+export function getTargetType(template: PipelineTemplateMetadata) {
+    if (template.attributes.deployTarget.toLowerCase().includes("webapp")) {
+        return TargetResourceType.WebApp;
+    } else if (template.attributes.deployTarget === "Azure:AKS") {
+        return TargetResourceType.AKS;
+    }
+    return TargetResourceType.None;
+}
+
+export function getTargetKind(template: PipelineTemplateMetadata) {
```

Comment:
add return type of function here as well.

Reply:
added